app/views/metrics/show.html.erb

```diff
@@ -2,15 +2,21 @@
 <h2>Page Data</h2>
 <h1><%= @summary.title %></h1>
 <div class="page-metadata">
-  <%= render :partial => 'metadata_row', locals: { label: 'Published', value: @summary.published_at } %>
-  <%= render :partial => 'metadata_row', locals: { label: 'Last updated', value: @summary.last_updated } %>
-  <%= render :partial => 'metadata_row', locals: {label: 'From', value:  @summary.publishing_organisation } %>
-  <%= render :partial => 'metadata_row', locals: {label: 'Type', value:  @summary.document_type } %>
-  <%= render :partial => 'metadata_row', locals: {label: 'URL', value:  @summary.base_path } %>
+  <%= render partial: 'metadata_row', locals: { label: 'Published', value: @summary.published_at } %>
```

Comment:
Not sure if we need `partial` and `locals`?

Reply:
Done